File: Nx3CS/bin/test/src/haxe/unit/TestStatus.cs

#pragma warning disable 109, 114, 219, 429, 168, 162
namespace haxe.unit{
	public  class TestStatus : global::haxe.lang.HxObject {
		public    TestStatus(global::haxe.lang.EmptyObject empty){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				{
				}
				
			}
			#line default
		}
		
		
		public    TestStatus(){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				global::haxe.unit.TestStatus.__hx_ctor_haxe_unit_TestStatus(this);
			}
			#line default
		}
		
		
		public static   void __hx_ctor_haxe_unit_TestStatus(global::haxe.unit.TestStatus __temp_me59){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				{
				}
				
			}
			#line default
		}
		
		
		public static  new object __hx_createEmpty(){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				return new global::haxe.unit.TestStatus(((global::haxe.lang.EmptyObject) (global::haxe.lang.EmptyObject.EMPTY) ));
			}
			#line default
		}
		
		
		public static  new object __hx_create(global::Array arr){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				return new global::haxe.unit.TestStatus();
			}
			#line default
		}
		
		
		public  bool done;
		
		public  bool success;
		
		public  string error;
		
		public  object posInfos;
		
		public override   double __hx_setField_f(string field, int hash, double @value, bool handleProperties){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				switch (hash){
					case 1512997393:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						this.posInfos = ((object) (@value) );
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return @value;
					}
					
					
					default:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return base.__hx_setField_f(field, hash, @value, handleProperties);
					}
					
				}
				
			}
			#line default
		}
		
		
		public override   object __hx_setField(string field, int hash, object @value, bool handleProperties){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				switch (hash){
					case 1512997393:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						this.posInfos = ((object) (@value) );
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return @value;
					}
					
					
					case 1932118984:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						this.error = global::haxe.lang.Runtime.toString(@value);
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return @value;
					}
					
					
					case 944645571:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						this.success = global::haxe.lang.Runtime.toBool(@value);
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return @value;
					}
					
					
					case 1114501250:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						this.done = global::haxe.lang.Runtime.toBool(@value);
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return @value;
					}
					
					
					default:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return base.__hx_setField(field, hash, @value, handleProperties);
					}
					
				}
				
			}
			#line default
		}
		
		
		public override   object __hx_getField(string field, int hash, bool throwErrors, bool isCheck, bool handleProperties){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				switch (hash){
					case 1512997393:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return this.posInfos;
					}
					
					
					case 1932118984:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return this.error;
					}
					
					
					case 944645571:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return this.success;
					}
					
					
					case 1114501250:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return this.done;
					}
					
					
					default:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return base.__hx_getField(field, hash, throwErrors, isCheck, handleProperties);
					}
					
				}
				
			}
			#line default
		}
		
		
		public override   double __hx_getField_f(string field, int hash, bool throwErrors, bool handleProperties){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				switch (hash){
					case 1512997393:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return ((double) (global::haxe.lang.Runtime.toDouble(this.posInfos)) );
					}
					
					
					default:
					{
						#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
						return base.__hx_getField_f(field, hash, throwErrors, handleProperties);
					}
					
				}
				
			}
			#line default
		}
		
		
		public override   void __hx_getFields(global::Array<object> baseArr){
			unchecked {
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				baseArr.push("posInfos");
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				baseArr.push("error");
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				baseArr.push("success");
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				baseArr.push("done");
				#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
				{
					#line 27 "F:\\HaxeToolkit\\haxe\\std\\haxe\\unit\\TestStatus.hx"
					base.__hx_getFields(baseArr);
				}
				
			}
			#line default
		}
		
		
	}
}

File: Nx3Java/java/src/haxe/unit/TestStatus.java
package haxe.unit;
import haxe.root.*;

@SuppressWarnings(value={"rawtypes", "unchecked"})
public  class TestStatus extends haxe.lang.HxObject
{
	public    TestStatus(haxe.lang.EmptyObject empty)
	{
		{
		}
		
	}
	
	
	public    TestStatus()
	{
		haxe.unit.TestStatus.__hx_ctor_haxe_unit_TestStatus(this);
	}
	
	
	public static   void __hx_ctor_haxe_unit_TestStatus(haxe.unit.TestStatus __temp_me51)
	{
		__temp_me51.done = false;
		__temp_me51.success = false;
	}
	
	
	public static   java.lang.Object __hx_createEmpty()
	{
		return new haxe.unit.TestStatus(((haxe.lang.EmptyObject) (haxe.lang.EmptyObject.EMPTY) ));
	}
	
	
	public static   java.lang.Object __hx_create(haxe.root.Array arr)
	{
		return new haxe.unit.TestStatus();
	}
	
	
	public  boolean done;
	
	public  boolean success;
	
	public  java.lang.String error;
	
	public  java.lang.String method;
	
	public  java.lang.String classname;
	
	public  java.lang.Object posInfos;
	
	public  java.lang.String backtrace;
	
	@Override public   double __hx_setField_f(java.lang.String field, double value, boolean handleProperties)
	{
		{
			boolean __temp_executeDef367 = true;
			switch (field.hashCode())
			{
				case 718070673:
				{
					if (field.equals("posInfos")) 
					{
						__temp_executeDef367 = false;
						this.posInfos = ((java.lang.Object) (value) );
						return value;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef367) 
			{
				return super.__hx_setField_f(field, value, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   java.lang.Object __hx_setField(java.lang.String field, java.lang.Object value, boolean handleProperties)
	{
		{
			boolean __temp_executeDef368 = true;
			switch (field.hashCode())
			{
				case 1354491070:
				{
					if (field.equals("backtrace")) 
					{
						__temp_executeDef368 = false;
						this.backtrace = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
				case 3089282:
				{
					if (field.equals("done")) 
					{
						__temp_executeDef368 = false;
						this.done = haxe.lang.Runtime.toBool(value);
						return value;
					}
					
					break;
				}
				
				
				case 718070673:
				{
					if (field.equals("posInfos")) 
					{
						__temp_executeDef368 = false;
						this.posInfos = ((java.lang.Object) (value) );
						return value;
					}
					
					break;
				}
				
				
				case -1867169789:
				{
					if (field.equals("success")) 
					{
						__temp_executeDef368 = false;
						this.success = haxe.lang.Runtime.toBool(value);
						return value;
					}
					
					break;
				}
				
				
				case -8935421:
				{
					if (field.equals("classname")) 
					{
						__temp_executeDef368 = false;
						this.classname = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
				case 96784904:
				{
					if (field.equals("error")) 
					{
						__temp_executeDef368 = false;
						this.error = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
				case -1077554975:
				{
					if (field.equals("method")) 
					{
						__temp_executeDef368 = false;
						this.method = haxe.lang.Runtime.toString(value);
						return value;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef368) 
			{
				return super.__hx_setField(field, value, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   java.lang.Object __hx_getField(java.lang.String field, boolean throwErrors, boolean isCheck, boolean handleProperties)
	{
		{
			boolean __temp_executeDef369 = true;
			switch (field.hashCode())
			{
				case 1354491070:
				{
					if (field.equals("backtrace")) 
					{
						__temp_executeDef369 = false;
						return this.backtrace;
					}
					
					break;
				}
				
				
				case 3089282:
				{
					if (field.equals("done")) 
					{
						__temp_executeDef369 = false;
						return this.done;
					}
					
					break;
				}
				
				
				case 718070673:
				{
					if (field.equals("posInfos")) 
					{
						__temp_executeDef369 = false;
						return this.posInfos;
					}
					
					break;
				}
				
				
				case -1867169789:
				{
					if (field.equals("success")) 
					{
						__temp_executeDef369 = false;
						return this.success;
					}
					
					break;
				}
				
				
				case -8935421:
				{
					if (field.equals("classname")) 
					{
						__temp_executeDef369 = false;
						return this.classname;
					}
					
					break;
				}
				
				
				case 96784904:
				{
					if (field.equals("error")) 
					{
						__temp_executeDef369 = false;
						return this.error;
					}
					
					break;
				}
				
				
				case -1077554975:
				{
					if (field.equals("method")) 
					{
						__temp_executeDef369 = false;
						return this.method;
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef369) 
			{
				return super.__hx_getField(field, throwErrors, isCheck, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   double __hx_getField_f(java.lang.String field, boolean throwErrors, boolean handleProperties)
	{
		{
			boolean __temp_executeDef370 = true;
			switch (field.hashCode())
			{
				case 718070673:
				{
					if (field.equals("posInfos")) 
					{
						__temp_executeDef370 = false;
						return ((double) (haxe.lang.Runtime.toDouble(this.posInfos)) );
					}
					
					break;
				}
				
				
			}
			
			if (__temp_executeDef370) 
			{
				return super.__hx_getField_f(field, throwErrors, handleProperties);
			}
			 else 
			{
				throw null;
			}
			
		}
		
	}
	
	
	@Override public   void __hx_getFields(haxe.root.Array<java.lang.String> baseArr)
	{
		baseArr.push("backtrace");
		baseArr.push("posInfos");
		baseArr.push("classname");
		baseArr.push("method");
		baseArr.push("error");
		baseArr.push("success");
		baseArr.push("done");
		{
			super.__hx_getFields(baseArr);
		}
		
	}
	
	
}


